components/ei-solutions/org.wso2.carbon.ei.solutions.ei.analytics/widgets/HorizontalBarChart/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>wso2sp-solutions.ei-analytics.horizontalbarchart</artifactId>
```

Comment:
Rename the widget with meaningful name

Reply:
Fixed with ( 8d4a31838382c2521640a73fbada3caf4c73101b 7b2df2b095d7ce643a2b920943ee88cfbab0d3bc 96d518077e89317005fe59ff8670a05d4f705b4b 9ae3d822e7addb1c50a2d7603489a4dd64be3180 07bb8f914fefa6f383f6b7ffce467290a322e356 d40163630eec02a0bee35aec75ef149842b4f45e 0df301308a5496995274a95eed4b528881a5e484 )